adflow/pyADflow.py

```diff
@@ -4135,6 +4178,26 @@ def computeJacobianVectorProductBwd(
         # Single return (most frequent) is 'clean', otherwise a tuple.
         return tuple(returns) if len(returns) > 1 else returns[0]
 
+    def computeJacobianVectorProductBwdFast(
+        self,
+        resBar=None,
+    ):
+        """used for testing the routines used for jac vec prods for the adjoint solve.
```

Comment:
Can you add more explanation here? "adjoint solve" is the type of language that does not really explain much while confusing newcomers. These fast routines are bwd-differentiated residual routines only with respect to the state and residuals themselves. We can also mention that this is the operator used for the matrix-free solution of the adjoint system.